package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ndustrial/contxt-sdk",
-  "version": "2.11.1",
+  "version": "2.11.2",
```

Comment:
Back this change out -- NPM does it automatically when you do `npm version`

(I'm not sure if you've published this yet so just pointing out the instructions in the README: https://github.com/ndustrialio/contxt-sdk-js#publishing)

Reply:
You've already done this :)